### PR TITLE
LPS-78779 Adjusting path

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test

**Comment by alinedoleron** [comment]
ci:reopen

**Comment by alinedoleron** [comment]
ci:test

**Comment by alinedoleron** [comment]
@pedroqueiroz, I believe these failures are not directly related to my fix.

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js [hunk 1]
```diff
@@ -340,7 +453,30 @@ AUI.add(
 
 					if (fieldName.match('-condition-first-operand')) {
 						var operatorSelected = instance._getOperator(index);
-						var type = instance._getDataType(field.getValue(), options);
+						var type;
```
Comment (pedroqueiroz) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js]: @alinedoleron , I think this is assigned but never used, can you double checked?

**Comment by pedroqueiroz** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alinedoleron** [comment]
ci:test

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js [hunk 2]
```diff
@@ -300,6 +394,25 @@ AUI.add(
 				return value || '';
 			},
 
+			_getType: function(value, options) {
+				var instance = this;
+
+				var option;
```
[the remark this answers is not in the document]
Comment (alinedoleron) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js]: Fixed

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js [hunk 3]
```diff
@@ -134,11 +198,35 @@ AUI.add(
 							);
 						}
 						else if (instance._isConstant(secondOperandTypeValue)) {
-							if (instance._getSecondOperandValue(index, 'input')) {
+							if (instance._getSecondOperandValue(index, 'date')) {
+								condition.operands.push(
+									{
+										type: instance._getFieldDataType(instance._getSecondOperandTypeValue(index)),
```
[the remark this answers is not in the document]
Comment (alinedoleron) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js]: Fixed

**Comment by pedroqueiroz** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alinedoleron** [comment]
ci:reopen

**Comment by alinedoleron** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**